dataland-inbound-proxy/Dockerfile

```diff
@@ -6,14 +6,15 @@ RUN apt-get update && apt-get install -y certbot cron && rm -rf /etc/nginx/* &&
 # Add CronJob for certificate renewal (Add 00:00 on every day)
 RUN echo "0 0 * * * root sh /scripts/renew-certificates.sh" > /etc/cron.d/certbot-renew
 
-COPY config /etc/nginx
-COPY templates/*-dataland-$PROXY_ENVIRONMENT.template /etc/nginx/templates/
-COPY templates/http-letsencrypt.template /etc/nginx/templates/http-letsencrypt.template
-COPY dhparam.pem /certs/dhparam.pem
-COPY docker-entrypoint.d/* /docker-entrypoint.d/
-COPY scripts /scripts
-COPY custom_errors.html /var/wwww/html/custom_errors.html
-COPY robots.txt /var/wwww/html/robots.txt
+COPY dataland-inbound-proxy/config /etc/nginx
```

Comment:
No the original version was ok.

But:  I have changed the context of the Dockerfile for the proxy in the docker-compose-file (see proxy: build: context: for the "proxy" service there).
The context is now one level higher, which forced me to add "dataland-inbound-proxy" to some paths here.

The reason why I set the context one level higher is, because now I need to copy stuff not only from the "dataland-inbound-proxy" folder into the container, but also from the "dataland-frontend" folder.
The additional COPY task copies the needed icons for the emails into the container, so that the proxy can serve them.

Reply:
Sure local location != server location